### DISCUSSION TITLE
Perguntas não técnicas para fazer na entrevista

### DISCUSSION THREAD

**Discussion by fabriciofmsilva** [frontendbr/forum#1178] [Discussão]
Amigos do fórum, dei uma pesquisada rápida aqui e não encontrei nada sobre. Gostaria de saber se você entrevista alguém quais as perguntas não ténicas que costuma fazer para os candidatos? Ou se você não entrevista, quais perguntas acha importante fazer?

**Comment by didiraja** [comment]
Não é o que você quer diretamente mas de uma olhada nas threads #917 e #911, porque movimentaram bastante o fórum e talvez você consiga absorver algo.

**Comment by fabriciofmsilva** [comment]
Valeu @didiraja, vou dar uma olhada!
